internal/manifests/config/build.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"bytes"
+	"embed"
+	"html/template"
```

Comment:
whats the difference to `text/template` ?

Reply:
Good question, I think the main difference is that html template will scape some characters